django/db/models/aggregates.py

```diff
@@ -62,12 +117,16 @@ class Count(Aggregate):
     name = 'Count'
     template = '%(function)s(%(distinct)s%(expressions)s)'
 
-    def __init__(self, expression, distinct=False, **extra):
+    def __init__(self, expression, distinct=False, filter=None, **extra):
         if expression == '*':
             expression = Star()
+
+        if isinstance(expression, Star) and filter is not None:
+            raise ValueError("Star cannot be used with filter. Please specify a field, e.g Count('id')")
```

Comment:
Dots are missing i.e.: `..., e.g. Count('id').`
On the other hand I think we can remove an example.